Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -172,7 +172,7 @@ defineSuite([
 
     it('rejects readyPromise with invalid tileset.json', function() {
         var tileset = scene.primitives.add(new Cesium3DTileset({
-            url : 'invalid'
+            url : 'invalid.json'
```

Comment:
This test should mock the loadJson function instead or relying on the server running the tests to actually fail.